winml/adapter/MLValueHelpers.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
what is this file for?

Reply:
great catch !

---
In reply to: [354039231](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354039231) [](ancestors = 354039231)